homeassistant/components/fireservicerota/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for FireServiceRota."""
+from pyfireservicerota import FireServiceRota, InvalidAuthError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_URL, CONF_USERNAME
+
+from .const import DOMAIN, URL_LIST  # pylint: disable=unused-import
+
+
+class FireServiceRotaFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a FireServiceRota config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+    DOMAIN = DOMAIN
+
+    def __init__(self):
+        """Initialize the config flow."""
+        self.data_schema = vol.Schema(
```

Comment:
This is a constant, so should make it `DATA_SCHEMA`

Reply:
Made it a constant.